chibuezeayogu/Library-App
# #163680633  Configure RuboCop

**What does the PR do?**
- This PR adds `RuboCop` rules to ensure that `RoR` style guide is adhered to.

**Description of Task to be completed?**
- Ensures that `RuboCop` is properly configured

**How should this be manually tested?**
- Clone the repository [Library-App](https://github.com/chibuezeayogu/Library-App.git) and
- Checkout to  this branch `ch-integrate-and-configure-rubocop-#163680633`
- run `rubocop` on the terminal
- ensure all `files are inspected and no offenses detected`

**What are the relevant pivotal tracker stories?**
- [#163680633](https://www.pivotaltracker.com/story/show/163680633)

**Screenshots**
- None
